crates/bitwarden/src/auth/client_auth.rs

```diff
@@ -37,14 +51,152 @@ impl<'a> ClientAuth<'a> {
         make_register_keys(email, password, kdf)
     }
 
+    pub async fn renew_token(&mut self) -> Result<()> {
+        renew_token(self.client).await
+    }
+
     #[cfg(feature = "internal")]
     pub async fn register(&mut self, input: &RegisterRequest) -> Result<()> {
         register(self.client, input).await
     }
+
+    #[cfg(feature = "internal")]
+    pub async fn prelogin(&mut self, email: String) -> Result<Kdf> {
+        use crate::auth::login::request_prelogin;
+
+        request_prelogin(self.client, email).await?.try_into()
+    }
+
+    #[cfg(feature = "internal")]
+    pub async fn login_password(
+        &mut self,
+        input: &PasswordLoginRequest,
```

Comment:
Do we need to rename the structs to match as well?

Reply:
We expose the structs externally, so I would prefer to not make that change right now.